app/src/routes/v1/fileTypes.js

```diff
@@ -0,0 +1,18 @@
+const fileTypesRouter = require('express').Router();
+const fileTypesComponent = require('../../components/fileTypes');
+
+/** Returns the dictionary of input/output file types */
+fileTypesRouter.get('/', async (_req, res, next) => {
+  
+  const fileTypes = await fileTypesComponent.getFileTypes();
```

Comment:
Based on what we know about that component, we don't need to do the async/await wrapper on this as that component (or soon to be JSON file resource) can be synchronous call. Can remove the async declaration on the callback function declaration on line 5 as well as there's no awaits in this function at all.